subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf5_txp.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(CONFIG_BT_CTLR_TX_PWR_PLUS_8)
+#define CONFIG_BT_CTLR_TX_PWR RADIO_TXPOWER_TXPOWER_Pos8dBm
```

Comment:
To avoid confusion, I'd use some other prefix than CONFIG_ here, since that's reserved for Kconfig symbols (which this is not)

Reply:
That might also help with finding references to undefined Kconfig symbols in source files, if that gets added.

Reply:
Looks like `drivers/usb/device/usb_dc_nrf5.c` uses `CFG_`.

Reply:
done.